lighthouse-core/audits/dobetterweb/uses-passive-event-listeners.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Audit a page to see if it is using passive event listeners on
+ * document-level event listeners (e.g. on window, document, document.body).
+ */
+
+'use strict';
+
+const url = require('url');
+const Audit = require('../audit');
+const Formatter = require('../../formatters/formatter');
+
+const SCROLL_BLOCKING_EVENTS = [
+  'wheel', 'mousewheel', 'touchstart', 'touchmove'
```

Comment:
wanna move these to a getBlockingEvents method on the audit so you dont have to duplicate in the test?